physical/file.go

```diff
@@ -48,10 +49,19 @@ func (b *FileBackend) Delete(path string) error {
 	b.l.Lock()
 	defer b.l.Unlock()
 
-	basePath, key := b.path(path)
-	fullPath := filepath.Join(basePath, key)
+	basePath, fileName := b.path(path)
+	fullPath := filepath.Join(basePath, "_"+fileName)
 
+	// For backwards compatibility, try to delete the file without base64 URL
```

Comment:
I think probably this should actually be the other way around; we should default to doing the faster thing for new installations and fall back if needed.